src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1127,6 +1127,122 @@ TIP: Since it runs a graphical application, the task will never exit and run for
 you want to check manually some results via the graphical application and then exit the application.
 The task will then terminate normally.
 
+== Workflow storage and versioning
+
+The Workflow Catalog provides the storage and versioning of workflows inside Buckets. It also provides a searching functionality using a simple query language to fetch workflows based on particular parameters. The Workflow Catalog is accessible via a REST API, and a http://swagger.io/[Swagger] interface is provided to help interacting with the API.
+
+=== Bucket concept
+
+A Bucket is a collection of Workflows that can be shared between users. The default settings provides the following Buckets:
+
+* Catalog
+* Templates (which is a set of workflow samples)
+
+=== Workflow versioning
+
+The Workflow Catalog provides versioning of workflows. Changes on an existing workflow will trigger the creation of a new version. By default, the catalog will always return the latest version of the workflows. It is possible to search for specific workflows using a query string.
```

Comment:
> Changes on an existing workflow will trigger the creation of a new version.

We discussed about that with Brian yesterday while reviewing the design document for the new Identity service. It's no longer true that a change will trigger the creation of a new version. Automatic save will  update the content of the latest revision. However, a new button will probably be added in the studio to offer the possibility to a user to explicitely create a revision.